src/tools/index.md

```diff
@@ -19,11 +19,10 @@ If you aren't sure which tools you need, **get the Flutter SDK.**
 
 [General-purpose tools]: #general-purpose-tools
 
-{{site.alert.version-note}}
-  As of Flutter 1.21, the Flutter SDK includes the full Dart SDK.
+{{site.alert.note}}
+The Flutter SDK includes the full Dart SDK.
```

Comment:
It's our standard to indent the note text 2 spaces. Idk why; I just obey.

```suggestion
  The Flutter SDK includes the full Dart SDK.
```

Reply:
Oops, thanks! Lost them in the copy paste :)